### ISSUE TITLE
Making all models compatible

### ISSUE THREAD

**Issue by stemangiola** [stemangiola/RNAseq-noise-model#6]
Hello @martinmodrak can I ask if you could please make the negBinomial_multinomial model compatible with the other that I made. Just for input variable and output variables (see logNormal, logStudent models).

Thanks!

**Comment by martinmodrak** [comment]
Yes, I am trying to figure out a good way to do this. Currently my models have some extra parameters (especially whether certain quantities are given or estimated) for development/debugging purposes. I think it however should be possible.

**Comment by stemangiola** [comment]
Cool, I meant nothing fancy just take the same data, and you see that now
lambda has a precise same meaning for all models, and generated quantities.
Nothing more.

Stefano Mangiola



*Data analyst (MSc, MPhil, PhD candidate) @ Melbourne University *

*Hovens lab - Papenfuss lab*
*Royal Melbourne Hospital - WEHI*
*+61 (0)450135954*


Il giorno mar 9 ott 2018 alle ore 20:16 Martin Modrák <
notifications@github.com> ha scritto:
